gym/spaces/dict.py

```diff
@@ -57,10 +64,10 @@ def __init__(self, spaces=None, seed=None, **spaces_kwargs):
                 space, Space
             ), "Values of the dict should be instances of gym.Space"
         super().__init__(
-            None, None, seed
+            None, None, seed  # type: ignore
         )  # None for shape and dtype, since it'll require special handling
 
-    def seed(self, seed=None):
+    def seed(self, seed: Optional[Union[t.Dict, int]] = None) -> list:
```

Comment:
For the return type here, I'd say either use `typing.List`, or commit to using `list`, `dict` and `tuple` everywhere. Since 3.7 (earliest we support), those can be made generic through `from __future__ import annotations`

Reply:
ok, will use native types whenever possible.